index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script src="https://kit.fontawesome.com/a3673060fd.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="styles.css" />
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>OSCARS-AWARDS-2023</title>
+</head>
+
+<body>
+  <header>
+    <div class="header">
+      <div class="head-cont">
+        <ul>
+          <li>
+            <a href="#"><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fab fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      <div class="desktop">
+        <div class="brand">
+          <a href="#home1"><img src="assests/logo/logo.png" alt="logo.img" /></a>
+        </div>
```

Comment:
- [ ] kindly consider adding a correct link to the home page on this logo so that if I am on the about page and I click on the logo being taken to the home page this will help you meet the below requirement 
> The logo contains a hyperlink to the homepage.